src/libraries/System.Threading.RateLimiting/src/System.Threading.RateLimiting.csproj

```diff
@@ -41,12 +42,13 @@ System.Threading.RateLimiting.RateLimitLease</PackageDescription>
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
```

Comment:
Is there any advantage on switching their locations here?
Also, if they are ItemGroups with the exact same condition, wouldn't it be ok to group both ItemGroups into a single one?

Reply:
Our convention is properties first, then compile and other items and last reference items. Keeping the reference items in a separate item group makes it easier to read a project's dependencies.